cinema/models.py

```diff
@@ -35,12 +40,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def uploads_movies(instance, filename):
+    _, extension = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}--{uuid.uuid4()}.{extension}"
```

Comment:
The `uploads_movies` function constructs the filename using `uuid.uuid4()`, but it should be `uuid.uuid4().hex` to ensure a proper string representation of the UUID. Additionally, the `extension` variable should not include the dot, so you might want to use `extension[1:]`.